src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/DataSourceSetup/eventWorkingListsDataSourceSetup.types.js

```diff
@@ -19,6 +24,6 @@ export type Props = {|
 export type EventWorkingListsDataSourceSetupOutputProps = {|
     ...RestProps,
     columns: EventWorkingListsColumnConfigs,
-    dataSource: DataSource,
+    dataSource?: DataSource,
```

Comment:
Relevant to the other comment I am making. `DataSource` will then not be optional. But will be annotated a little bit different 

```
export type DataSource = Array<?DataSourceItem>;
```

I dont know your opinion on this but for me defining the value with _same type_  instead of `null` is more consistent. I guess I dont have any knockout arguments but perhaps is something we can think of.

Reply:
To be clear, It's never `null` here, it's either `undefined` or it has value. I don't feel it makes sense to change it in this instance.